bokeh/charts/utils.py

```diff
@@ -34,8 +34,11 @@
 #-----------------------------------------------------------------------------
 
 # TODO: (bev) this should go in a plotting utils one level up
+# TODO: is this used vs. the DEFAULT_PALETTE in _builder.py ?
```

Comment:
IIRC it's used when the palette is not specific in `cycle_colors` while the one in _builder.py is used as a default chart argument when `cycle_colors` is called from a chart...